.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+cache:
+  directories:
+    - $HOME/.m2
+language: java
+jdk:
+  - oraclejdk8
+script:
+  - '[ "${TRAVIS_PULL_REQUEST}" != "false" ] || ./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'
```

Comment:
@ilayaperumalg, 

please, take a look into the Travis report.
Is that really an expected behavior?
I would like to see some logs about building project, but there is no.

Maybe we don't need this `TRAVIS_PULL_REQUEST` condition at all?